docs/org/reference/language/functions.org

```diff
@@ -37,13 +37,52 @@ Function declarations can depend on each other recursively. In the following exa
 #+begin_example
 open import Stdlib.Prelude;
 
-odd : Nat → Bool;
-
-even : Nat → Bool;
+odd : Nat -> Bool;
+even : Nat -> Bool;
 
 odd zero := false;
 odd (suc n) := even n;
 
 even zero := true;
 even (suc n) := odd n;
 #+end_example
+
+** Anonymous functions
+
+Anonymous functions, or /lambdas/, are introduced with the syntax:
+
+#+begin_example
+\{| pat1 .. patN_1 := clause1
+  | ..
+  | pat1 .. patN_M := clauseM}
+#+end_example
+
+The first pipe =|= is optional. Instead of =\= one can also use =λ=.
+
+An anonymous function just lists all clauses of a function without
+naming it. Any function declaration can be converted to use anonymous
+functions:
+
+#+begin_example
+open import Stdlib.Prelude;
+
+odd : Nat -> Bool;
+even : Nat -> Bool;
+
+odd := \{
+  zero := false;
+  (suc n) := even n;
+};
+
+even := \{
+  zero := true;
+  (suc n) := odd n;
+};
+#+end_example
+
+** Short definitions
+
+If a function has only one clause, then its definition can written in one line, with the body immediately following the signature:
```

Comment:
Please reformulate this sentence. Also, consider that if I have this function with two clauses:

```
  giveMeTheTruth : Nat -> Bool := λ {
      | zero := true
      | (suc n) := false
    };
```
I can also write in one line `giveMeTheTruth : Nat -> Bool := λ { | zero := true | (suc n):= false };`

Reply:
Well, this function has one clause whose body is a lambda with two clauses. How else can express the fact that there is only one clause in the function *definition*? I just removed any mention of clauses, if it's confusing.

Reply:
Yes, it was a bit confusing.
You still have a typo. It should be "can be written"

Reply:
I already fixed the typo.

Reply:
Thanks. I see it now.